[2022-12-31T20:51:31.705+0700] {processor.py:153} INFO - Started process (PID=10630) to work on /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:51:31.706+0700] {processor.py:743} INFO - Processing file /home/dmk/airflow/dags/example_twitter.py for tasks to queue
[2022-12-31T20:51:31.707+0700] {logging_mixin.py:137} INFO - [2022-12-31T20:51:31.707+0700] {dagbag.py:538} INFO - Filling up the DagBag from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:51:31.718+0700] {logging_mixin.py:137} INFO - [2022-12-31T20:51:31.717+0700] {dagbag.py:343} ERROR - Failed to import: /home/dmk/airflow/dags/example_twitter.py
Traceback (most recent call last):
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/dmk/airflow/dags/example_twitter.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/home/dmk/anaconda3/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 984, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.apache'
[2022-12-31T20:51:31.719+0700] {processor.py:755} WARNING - No viable dags retrieved from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:51:31.741+0700] {processor.py:175} INFO - Processing /home/dmk/airflow/dags/example_twitter.py took 0.040 seconds
[2022-12-31T20:54:09.231+0700] {processor.py:153} INFO - Started process (PID=11579) to work on /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:54:09.232+0700] {processor.py:743} INFO - Processing file /home/dmk/airflow/dags/example_twitter.py for tasks to queue
[2022-12-31T20:54:09.233+0700] {logging_mixin.py:137} INFO - [2022-12-31T20:54:09.232+0700] {dagbag.py:538} INFO - Filling up the DagBag from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:54:09.241+0700] {logging_mixin.py:137} INFO - [2022-12-31T20:54:09.240+0700] {dagbag.py:343} ERROR - Failed to import: /home/dmk/airflow/dags/example_twitter.py
Traceback (most recent call last):
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/dmk/airflow/dags/example_twitter.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/home/dmk/anaconda3/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 984, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.apache.hive'
[2022-12-31T20:54:09.242+0700] {processor.py:755} WARNING - No viable dags retrieved from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:54:09.263+0700] {processor.py:175} INFO - Processing /home/dmk/airflow/dags/example_twitter.py took 0.034 seconds
[2022-12-31T20:55:28.687+0700] {processor.py:153} INFO - Started process (PID=12004) to work on /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:55:28.690+0700] {processor.py:743} INFO - Processing file /home/dmk/airflow/dags/example_twitter.py for tasks to queue
[2022-12-31T20:55:28.691+0700] {logging_mixin.py:137} INFO - [2022-12-31T20:55:28.691+0700] {dagbag.py:538} INFO - Filling up the DagBag from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:55:28.700+0700] {logging_mixin.py:137} INFO - [2022-12-31T20:55:28.699+0700] {dagbag.py:343} ERROR - Failed to import: /home/dmk/airflow/dags/example_twitter.py
Traceback (most recent call last):
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/dmk/airflow/dags/example_twitter.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/home/dmk/anaconda3/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 984, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.apache.hive'
[2022-12-31T20:55:28.701+0700] {processor.py:755} WARNING - No viable dags retrieved from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:55:28.727+0700] {processor.py:175} INFO - Processing /home/dmk/airflow/dags/example_twitter.py took 0.044 seconds
[2022-12-31T20:55:47.177+0700] {processor.py:153} INFO - Started process (PID=12076) to work on /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:55:47.178+0700] {processor.py:743} INFO - Processing file /home/dmk/airflow/dags/example_twitter.py for tasks to queue
[2022-12-31T20:55:47.179+0700] {logging_mixin.py:137} INFO - [2022-12-31T20:55:47.178+0700] {dagbag.py:538} INFO - Filling up the DagBag from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:55:47.189+0700] {logging_mixin.py:137} INFO - [2022-12-31T20:55:47.188+0700] {dagbag.py:343} ERROR - Failed to import: /home/dmk/airflow/dags/example_twitter.py
Traceback (most recent call last):
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/dmk/airflow/dags/example_twitter.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/home/dmk/anaconda3/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 984, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.apache.hive'
[2022-12-31T20:55:47.190+0700] {processor.py:755} WARNING - No viable dags retrieved from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:55:47.208+0700] {processor.py:175} INFO - Processing /home/dmk/airflow/dags/example_twitter.py took 0.033 seconds
[2022-12-31T20:56:11.047+0700] {processor.py:153} INFO - Started process (PID=12170) to work on /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:56:11.049+0700] {processor.py:743} INFO - Processing file /home/dmk/airflow/dags/example_twitter.py for tasks to queue
[2022-12-31T20:56:11.050+0700] {logging_mixin.py:137} INFO - [2022-12-31T20:56:11.050+0700] {dagbag.py:538} INFO - Filling up the DagBag from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:56:11.059+0700] {logging_mixin.py:137} INFO - [2022-12-31T20:56:11.058+0700] {dagbag.py:343} ERROR - Failed to import: /home/dmk/airflow/dags/example_twitter.py
Traceback (most recent call last):
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/dmk/airflow/dags/example_twitter.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/home/dmk/anaconda3/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 984, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.apache.hive'
[2022-12-31T20:56:11.060+0700] {processor.py:755} WARNING - No viable dags retrieved from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:56:11.079+0700] {processor.py:175} INFO - Processing /home/dmk/airflow/dags/example_twitter.py took 0.034 seconds
[2022-12-31T20:56:37.379+0700] {processor.py:153} INFO - Started process (PID=12253) to work on /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:56:37.380+0700] {processor.py:743} INFO - Processing file /home/dmk/airflow/dags/example_twitter.py for tasks to queue
[2022-12-31T20:56:37.381+0700] {logging_mixin.py:137} INFO - [2022-12-31T20:56:37.381+0700] {dagbag.py:538} INFO - Filling up the DagBag from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:56:37.390+0700] {logging_mixin.py:137} INFO - [2022-12-31T20:56:37.389+0700] {dagbag.py:343} ERROR - Failed to import: /home/dmk/airflow/dags/example_twitter.py
Traceback (most recent call last):
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/dmk/airflow/dags/example_twitter.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/home/dmk/anaconda3/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 984, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.apache.hive'
[2022-12-31T20:56:37.390+0700] {processor.py:755} WARNING - No viable dags retrieved from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:56:37.412+0700] {processor.py:175} INFO - Processing /home/dmk/airflow/dags/example_twitter.py took 0.035 seconds
[2022-12-31T20:57:04.343+0700] {processor.py:153} INFO - Started process (PID=12432) to work on /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:57:04.344+0700] {processor.py:743} INFO - Processing file /home/dmk/airflow/dags/example_twitter.py for tasks to queue
[2022-12-31T20:57:04.344+0700] {logging_mixin.py:137} INFO - [2022-12-31T20:57:04.344+0700] {dagbag.py:538} INFO - Filling up the DagBag from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:57:04.353+0700] {logging_mixin.py:137} INFO - [2022-12-31T20:57:04.352+0700] {dagbag.py:343} ERROR - Failed to import: /home/dmk/airflow/dags/example_twitter.py
Traceback (most recent call last):
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/dmk/airflow/dags/example_twitter.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/home/dmk/anaconda3/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 984, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.apache.hive'
[2022-12-31T20:57:04.354+0700] {processor.py:755} WARNING - No viable dags retrieved from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:57:04.370+0700] {processor.py:175} INFO - Processing /home/dmk/airflow/dags/example_twitter.py took 0.030 seconds
[2022-12-31T20:57:41.511+0700] {processor.py:153} INFO - Started process (PID=12687) to work on /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:57:41.516+0700] {processor.py:743} INFO - Processing file /home/dmk/airflow/dags/example_twitter.py for tasks to queue
[2022-12-31T20:57:41.517+0700] {logging_mixin.py:137} INFO - [2022-12-31T20:57:41.517+0700] {dagbag.py:538} INFO - Filling up the DagBag from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:57:41.528+0700] {logging_mixin.py:137} INFO - [2022-12-31T20:57:41.527+0700] {dagbag.py:343} ERROR - Failed to import: /home/dmk/airflow/dags/example_twitter.py
Traceback (most recent call last):
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/dmk/airflow/dags/example_twitter.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/home/dmk/anaconda3/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 984, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.apache.hive'
[2022-12-31T20:57:41.529+0700] {processor.py:755} WARNING - No viable dags retrieved from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:57:41.548+0700] {processor.py:175} INFO - Processing /home/dmk/airflow/dags/example_twitter.py took 0.041 seconds
[2022-12-31T20:58:07.299+0700] {processor.py:153} INFO - Started process (PID=12835) to work on /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:58:07.300+0700] {processor.py:743} INFO - Processing file /home/dmk/airflow/dags/example_twitter.py for tasks to queue
[2022-12-31T20:58:07.301+0700] {logging_mixin.py:137} INFO - [2022-12-31T20:58:07.300+0700] {dagbag.py:538} INFO - Filling up the DagBag from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:58:07.310+0700] {logging_mixin.py:137} INFO - [2022-12-31T20:58:07.309+0700] {dagbag.py:343} ERROR - Failed to import: /home/dmk/airflow/dags/example_twitter.py
Traceback (most recent call last):
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/dmk/airflow/dags/example_twitter.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/home/dmk/anaconda3/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 984, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.apache.hive'
[2022-12-31T20:58:07.310+0700] {processor.py:755} WARNING - No viable dags retrieved from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:58:07.325+0700] {processor.py:175} INFO - Processing /home/dmk/airflow/dags/example_twitter.py took 0.029 seconds
[2022-12-31T20:58:32.548+0700] {processor.py:153} INFO - Started process (PID=12959) to work on /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:58:32.552+0700] {processor.py:743} INFO - Processing file /home/dmk/airflow/dags/example_twitter.py for tasks to queue
[2022-12-31T20:58:32.552+0700] {logging_mixin.py:137} INFO - [2022-12-31T20:58:32.552+0700] {dagbag.py:538} INFO - Filling up the DagBag from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:58:32.657+0700] {logging_mixin.py:137} INFO - [2022-12-31T20:58:32.656+0700] {dagbag.py:343} ERROR - Failed to import: /home/dmk/airflow/dags/example_twitter.py
Traceback (most recent call last):
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/dmk/airflow/dags/example_twitter.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/home/dmk/anaconda3/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 984, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.apache.hive'
[2022-12-31T20:58:32.657+0700] {processor.py:755} WARNING - No viable dags retrieved from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:58:32.672+0700] {processor.py:175} INFO - Processing /home/dmk/airflow/dags/example_twitter.py took 0.127 seconds
[2022-12-31T20:58:58.897+0700] {processor.py:153} INFO - Started process (PID=13084) to work on /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:58:58.897+0700] {processor.py:743} INFO - Processing file /home/dmk/airflow/dags/example_twitter.py for tasks to queue
[2022-12-31T20:58:58.898+0700] {logging_mixin.py:137} INFO - [2022-12-31T20:58:58.898+0700] {dagbag.py:538} INFO - Filling up the DagBag from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:58:58.904+0700] {logging_mixin.py:137} INFO - [2022-12-31T20:58:58.903+0700] {dagbag.py:343} ERROR - Failed to import: /home/dmk/airflow/dags/example_twitter.py
Traceback (most recent call last):
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/dmk/airflow/dags/example_twitter.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/home/dmk/anaconda3/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 984, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.apache.hive'
[2022-12-31T20:58:58.905+0700] {processor.py:755} WARNING - No viable dags retrieved from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:58:58.923+0700] {processor.py:175} INFO - Processing /home/dmk/airflow/dags/example_twitter.py took 0.125 seconds
[2022-12-31T20:59:22.882+0700] {processor.py:153} INFO - Started process (PID=13321) to work on /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:59:22.883+0700] {processor.py:743} INFO - Processing file /home/dmk/airflow/dags/example_twitter.py for tasks to queue
[2022-12-31T20:59:22.884+0700] {logging_mixin.py:137} INFO - [2022-12-31T20:59:22.884+0700] {dagbag.py:538} INFO - Filling up the DagBag from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:59:22.895+0700] {logging_mixin.py:137} INFO - [2022-12-31T20:59:22.893+0700] {dagbag.py:343} ERROR - Failed to import: /home/dmk/airflow/dags/example_twitter.py
Traceback (most recent call last):
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/dmk/airflow/dags/example_twitter.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/home/dmk/anaconda3/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 984, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.apache.hive'
[2022-12-31T20:59:22.895+0700] {processor.py:755} WARNING - No viable dags retrieved from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:59:22.918+0700] {processor.py:175} INFO - Processing /home/dmk/airflow/dags/example_twitter.py took 0.039 seconds
[2022-12-31T20:59:44.874+0700] {processor.py:153} INFO - Started process (PID=14235) to work on /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:59:44.875+0700] {processor.py:743} INFO - Processing file /home/dmk/airflow/dags/example_twitter.py for tasks to queue
[2022-12-31T20:59:44.876+0700] {logging_mixin.py:137} INFO - [2022-12-31T20:59:44.875+0700] {dagbag.py:538} INFO - Filling up the DagBag from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:59:44.885+0700] {logging_mixin.py:137} INFO - [2022-12-31T20:59:44.884+0700] {dagbag.py:343} ERROR - Failed to import: /home/dmk/airflow/dags/example_twitter.py
Traceback (most recent call last):
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/dmk/airflow/dags/example_twitter.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/home/dmk/anaconda3/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 984, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.apache.hive'
[2022-12-31T20:59:44.885+0700] {processor.py:755} WARNING - No viable dags retrieved from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T20:59:44.903+0700] {processor.py:175} INFO - Processing /home/dmk/airflow/dags/example_twitter.py took 0.034 seconds
[2022-12-31T21:00:16.532+0700] {processor.py:153} INFO - Started process (PID=15087) to work on /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T21:00:16.537+0700] {processor.py:743} INFO - Processing file /home/dmk/airflow/dags/example_twitter.py for tasks to queue
[2022-12-31T21:00:16.539+0700] {logging_mixin.py:137} INFO - [2022-12-31T21:00:16.539+0700] {dagbag.py:538} INFO - Filling up the DagBag from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T21:00:16.560+0700] {logging_mixin.py:137} INFO - [2022-12-31T21:00:16.557+0700] {dagbag.py:343} ERROR - Failed to import: /home/dmk/airflow/dags/example_twitter.py
Traceback (most recent call last):
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/dmk/airflow/dags/example_twitter.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/home/dmk/anaconda3/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 984, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.apache.hive'
[2022-12-31T21:00:16.561+0700] {processor.py:755} WARNING - No viable dags retrieved from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T21:00:16.588+0700] {processor.py:175} INFO - Processing /home/dmk/airflow/dags/example_twitter.py took 0.061 seconds
[2022-12-31T21:00:43.744+0700] {processor.py:153} INFO - Started process (PID=15442) to work on /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T21:00:43.744+0700] {processor.py:743} INFO - Processing file /home/dmk/airflow/dags/example_twitter.py for tasks to queue
[2022-12-31T21:00:43.745+0700] {logging_mixin.py:137} INFO - [2022-12-31T21:00:43.745+0700] {dagbag.py:538} INFO - Filling up the DagBag from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T21:00:43.758+0700] {logging_mixin.py:137} INFO - [2022-12-31T21:00:43.756+0700] {dagbag.py:343} ERROR - Failed to import: /home/dmk/airflow/dags/example_twitter.py
Traceback (most recent call last):
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/dmk/airflow/dags/example_twitter.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/home/dmk/anaconda3/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 984, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.apache.hive'
[2022-12-31T21:00:43.758+0700] {processor.py:755} WARNING - No viable dags retrieved from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T21:00:43.776+0700] {processor.py:175} INFO - Processing /home/dmk/airflow/dags/example_twitter.py took 0.035 seconds
[2022-12-31T21:01:03.164+0700] {processor.py:153} INFO - Started process (PID=15735) to work on /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T21:01:03.168+0700] {processor.py:743} INFO - Processing file /home/dmk/airflow/dags/example_twitter.py for tasks to queue
[2022-12-31T21:01:03.169+0700] {logging_mixin.py:137} INFO - [2022-12-31T21:01:03.169+0700] {dagbag.py:538} INFO - Filling up the DagBag from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T21:01:03.179+0700] {logging_mixin.py:137} INFO - [2022-12-31T21:01:03.178+0700] {dagbag.py:343} ERROR - Failed to import: /home/dmk/airflow/dags/example_twitter.py
Traceback (most recent call last):
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/models/dagbag.py", line 339, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/dmk/airflow/dags/example_twitter.py", line 34, in <module>
    from airflow.operators.hive_operator import HiveOperator
  File "/home/dmk/anaconda3/lib/python3.9/site-packages/airflow/utils/deprecation_tools.py", line 36, in getattr_with_deprecation
    return getattr(importlib.import_module(new_module), new_class_name)
  File "/home/dmk/anaconda3/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 972, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 984, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'airflow.providers.apache.hive'
[2022-12-31T21:01:03.180+0700] {processor.py:755} WARNING - No viable dags retrieved from /home/dmk/airflow/dags/example_twitter.py
[2022-12-31T21:01:03.195+0700] {processor.py:175} INFO - Processing /home/dmk/airflow/dags/example_twitter.py took 0.037 seconds
